app/controllers/lti_controller.rb

```diff
@@ -0,0 +1,81 @@
+class LtiController < ApplicationController
+  after_action :allow_iframe, only: [:launch]
+  
+  def launch
+    #If set, then hide the header and footer
+    session[:isLTI]=true
+
+    #check to see if the LTI post request from TC has a consumer key
+    if not Rails.configuration.lti_settings[params[:oauth_consumer_key]]
+      render :launch_error, status: 401
+      return
+    end
+
+
+    require 'oauth/request_proxy/action_controller_request'
```

Comment:
my issues start with line 15 -- getting this error:
LoadError at /lti/launch
cannot load such file -- oauth/request_proxy/action_controller_request

Reply:
figured this one out -- looks like I needed to use an earlier version o the gem lms-lti